src/components/Navbar/Navbar.tsx

```diff
@@ -3,10 +3,15 @@ import CFPBLogo from '../../assets/images/cfpb-logo.png';
 import Link from '../Link/Link';
 import './navbar.less';
 
-export function CfpbLogo(): JSX.Element {
+interface CfpbLogoProperties {
+  href?: string;
+}
+
+export function CfpbLogo({ href }: CfpbLogoProperties): JSX.Element {
```

Comment:
Instead of relying on nullish coalescing on line 14, we usually prefer declaring default function parameters for our props where we can. Maybe revert the line 14 change and change line 10 to be...
```suggestion
export function CfpbLogo({
  href = 'https://www.consumerfinance.gov'
}: CfpbLogoProperties): JSX.Element {
```

Reply:
Updated. Thank you.

Reply:
Looks good! 👍